### ISSUE TITLE
Incorrect indentation notice for anonymous classes

### ISSUE THREAD

**Issue by kwn** [squizlabs/PHP_CodeSniffer#884]
Hi, I've just find out, that Code Sniffer treat this as an issue:

```php
    public function setUp()
    {
        $this->dto = new class {
            public $name = 'Some value';
        };
        $this->normalizedDto = ['name' => 'Some value'];

        // [...]
    }
```
Error message:

```
 31 | ERROR | [x] Line indented incorrectly; expected 8 spaces, found
    |       |     12
```

Line no 31 is the one, with `public $name = 'Some value';`. I believe all properties and methods in anonymous class should be indented, and it shouldn't be reported as an error.

**Comment by gsherwood** [comment]
The problem was that anon classes get their own PHPCS-assigned token and this token wasn't in the list of scope openers. So the indentation sniff didn't see the class as opening a new scope and thus requiring more indentation. It now does. Thanks for reporting.

**Comment by kwn** [comment]
Sweet! Thanks for fixing that :+1:
